.github/workflows/container-image.yml

```diff
@@ -12,7 +12,7 @@ on:
   workflow_dispatch: {}
 
 env:
-  CONTAINER_CMD: docker
+  CONTAINER_CMD: podman
```

Comment:
I agree here, and we should probably align to `podman` in all of samba-in-kubernetes sub-projects. However, it would be nice to add to the `checks` stage something like `run: command -v ${{ env.CONTAINER_CMD }}`